update_prevalence.py

```diff
@@ -621,13 +621,29 @@ def main() -> None:
                 if (
                     place.population == 0
                     and place.name not in ("Unassigned", "Unknown")
+                    and place.name != "New York City"  # Hack: see below
                 ):
                     raise ValueError(
                         f"JHU data has cases but no population for {place!r}"
                     )
                 place.cumulative_cases[current] = line.Confirmed
             current -= timedelta(days=1)
 
+        # Hack: On 2020-09-01, JHU switched from reporting all NYC cases
+        # under a virtual county called "New York City" to reporting them
+        # under the counties for individual boroughs. Until 2020-09-14, we'll
+        # support mixing the pre- and post-transition data by merging the
+        # five borough counties (New York, Kings, Queens, Richmond, Bronx)
+        # into "New York City".
+        nyc_county_names = ("New York", "Kings", "Queens", "Richmond", "Bronx")
+        ny_state = data.countries["US"].states["New York"]
+        nyc_indv = [ny_state.counties.pop(cname) for cname in nyc_county_names]
```

Comment:
Doesn't pop() remove the county from the dict? The above comment seems to imply that this week is meant as a transition in which we should expose the county data.

Also... why support New York City as a county for a limited amount of time? Seems if you're going to drop it in the future, why not now?

Reply:
Before 2020-09-01, we only have aggregated data for NYC. After 2020-09-01, we only have split-out data per borough. Thus, the only thing we can report for a time interval that spans the transition is the aggregated data. Thus, pop() to remove the individual borough counties is correct. (Otherwise, you could select an individual borough in the dropdown but it wouldn't have a full week's worth of cases.)

Reply:
Ah, ok, so this code will be removed on the 14th? Could you put in an early return on that date so that this can't get rolled forward by accident?